packages/ckeditor5-dev-tests/lib/tasks/runmanualtests.js

```diff
@@ -144,6 +144,8 @@ module.exports = function runManualTests( options ) {
 				await buildDllInRepository( path.dirname( pkgJsonPath ) );
 			}
 		}
+
+		console.log( '\n📍 DLL building complete!\n' );
```

Comment:
```suggestion
		console.log( '\n📍 DLL building complete.\n' );
```

There is no need to shout :D